ddtrace/contrib/trace_utils.py

```diff
@@ -415,3 +416,33 @@ def set_flattened_tags(
     for prefix, value in items:
         for tag, v in _flatten(value, sep, prefix, exclude_policy):
             span.set_tag(tag, processor(v) if processor is not None else v)
+
+
+def set_user(tracer, user_id, name="", email="", scope="", role="", session_id=""):
+    # type: (Tracer, str, str, str, str, str) -> None
```

Comment:
```suggestion
def set_user(tracer, user_id, name=None, email="", scope="", role="", session_id=""):
    # type: (Tracer, Optional[str], str, str, str, str) -> None
```

should we use `None`/`Optional[str]` as the default values/types for all of these parameters?